plugins/techdocs-backend/src/search/DefaultTechDocsCollator.ts

```diff
@@ -0,0 +1,148 @@
+/*
+ * Copyright 2021 The Backstage Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import { PluginEndpointDiscovery } from '@backstage/backend-common';
+import { Entity, RELATION_OWNED_BY } from '@backstage/catalog-model';
+import { IndexableDocument, DocumentCollator } from '@backstage/search-common';
+import fetch from 'cross-fetch';
+import unescape from 'lodash/unescape';
+import { Logger } from 'winston';
+import pLimit from 'p-limit';
+import { CatalogApi, CatalogClient } from '@backstage/catalog-client';
+
+export interface MkSearchIndexDoc {
+  title: string;
+  text: string;
+  location: string;
+}
+
+export interface TechDocsDocument extends IndexableDocument {
+  kind: string;
+  namespace: string;
+  entity: string;
```

Comment:
```suggestion
  name: string;
```
I wonder why this isn't detected